ui/statistics_page.go

```diff
@@ -0,0 +1,120 @@
+package ui
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"gioui.org/layout"
+	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/values"
+	"github.com/planetdecred/godcr/wallet"
+)
+
+const PageStat = "Stat"
+
+type statPage struct {
+	common      pageCommon
+	txs         **wallet.Transactions
+	theme       *decredmaterial.Theme
+	l           layout.List
+	startupTime string
+	syncStatus  *wallet.SyncStatus
+}
+
+func (win *Window) StatPage(common pageCommon) Page {
+	pg := &statPage{
+		txs:    &win.walletTransactions,
+		common: common,
+		theme:  common.theme,
+		l: layout.List{
+			Axis: layout.Vertical,
+		},
+	}
+	pg.startupTime = time.Now().String()
+	pg.syncStatus = win.walletSyncStatus
+
+	return pg
+}
+
+func (pg *statPage) lineSeparator() layout.Widget {
+	return func(gtx C) D {
+		return pg.theme.Separator().Layout(gtx)
+	}
+}
+
+func (pg *statPage) layoutStats(gtx C) D {
+	background := pg.common.theme.Color.Surface
+	card := pg.common.theme.Card()
+	card.Color = background
+	inset := layout.Inset{
+		Top:    values.MarginPadding12,
+		Bottom: values.MarginPadding12,
+		Right:  values.MarginPadding16,
+		Left:   values.MarginPadding16,
+	}
```

Comment:
should the values need be updated at any point, you have have one point of edit.
```suggestion
       mtb := values.MarginPadding12,
       mlr := values.MarginPadding16,
	inset := layout.Inset{
		Top:    mtb,
		Bottom: mtb,
		Right:  mrl,
		Left:   mrl,
	}
```

Reply:
I don't think this change is necessary. The reason for the values package is already to be able to manage values from a single point if there's need for it. This is a local scope and the variable declaration to its use is pretty close, so it's not necessary. Besides the former has shorter lines of code than the suggestion.